src/Core/Components/Switch/FluentSwitch.razor

```diff
@@ -2,7 +2,7 @@
 @using Microsoft.FluentUI.AspNetCore.Components.Extensions
 @inherits FluentInputBase<bool>
 
-<FluentField InputComponent="@this" ForId="@Id" Class="@ClassValue" Style="@((ReadOnly ? "pointer-events: none;" : "") + StyleValue)">
```

Comment:
`AdditionalAttributes` is a the `fluent-switch` attribute. You therefore need to check whether **tabindex** is already in it, because if dev specifies a value for this attribute, you can't have it twice in your component. If `ReadOnly` is activated and there is no tabindex in this list, then you need to add `tabindex="-1"`, otherwise you can use the keyboard to access the component. The pointer-events style is for mouse click only. Go to `FluentAvatar.razor.cs` where this has already been done.

Reply:
`GetTabIndexValue` as in `FluentAvatar.razor.cs`